addons/mail/security/mail_security.xml

```diff
@@ -273,4 +273,23 @@
             <field name="perm_create" eval="True"/>
             <field name="perm_unlink" eval="True"/>
         </record>
+
+        <record id="mail_canned_response_user_rule" model="ir.rule">
```

Comment:
Beware guidelines ! (and cwu for create / write / unlink, as far as I understand those rules)
```suggestion
        <record id="ir_rule_mail_canned_response_user_cwu" model="ir.rule">
```